R/helpers-ppc.R

```diff
@@ -250,6 +276,203 @@ all_counts <- function(x, ...) {
   all_whole_number(x, ...) && min(x) >= 0
 }
 
+#' Obtain the coverage parameter for simultaneous confidence bands for ECDFs
+#'
+#' @param N Length of sample.
+#' @param L Number of chains. Used for MCMC, defaults to 1 for ppc.
+#' @param K Number of equally spaced evaluation points (1:K / K). Defaults to N.
+#' @param prob Desired simultaneous coverage (0,1).
+#' @param M number of simulations to run, if simulation method is used.
+#' @param adj_method String defining the desired adjustment method. By default
```

Comment:
adj_method is documented but interpolate_adj ist given as the actual argument

Reply:
The 'adj_method' parameter was a remnant of the earlier version of the function. I have now updated the documentation to correctly refer to 'interpolate_adj'.